src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1954,8 +1954,8 @@
     <Field Name="StringEndToken" Type="SyntaxToken">
       <Kind Name="InterpolatedStringEndToken"/>
       <Kind Name="InterpolatedStringEndToken"/>
-      <Kind Name="InterpolatedSingleLineRawStringEndToken"/>
-      <Kind Name="InterpolatedMultiLineRawStringEndToken"/>
+      <Kind Name="InterpolatedRawStringEndToken"/>
+      <Kind Name="InterpolatedRawStringEndToken"/>
```

Comment:
The duplication looks unintentional

Reply:
it is intentional.  teh two non-raw forms are terminated by an InterpolatedStringEndToken.  The two raw forms are both terminated by a InterpolatedRawStringEndToken

Reply:
I didn't follow. What happens if we de-dupe (both for raw and for non-raw)?
As far as I can tell, this duplication has no impact on generated syntax code.

Reply:
Resolved offline (this affects the g4 generation)